src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
+
+.section_1_img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 36px;
+}
+
+.section_1__text {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  top: 175px;
+  margin: 0;
+  color: #060b35;
+  line-height: 18px;
+  font-family: Roboto, sans-serif;
+  text-align: left;
+}
+
+.section_1__code {
+  text-align: left;
+  height: 14px;
+  width: 98px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  font-weight: 400;
+  margin: 6px 0 3px;
+}
```

Comment:
The class name '.section_1__code' follows the BEM naming convention assuming 'section_1' is the block and 'code' is the element. However, ensure that 'section_1' is indeed the block name as per your project's structure.